opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/util/logresolver/Constants.java

```diff
@@ -0,0 +1,19 @@
+package de.adorsys.opba.protocol.facade.util.logresolver;
+
+import lombok.experimental.UtilityClass;
+
+
+/**
+ * LogResolver constant names.
+ */
+
+
+@UtilityClass
+@SuppressWarnings("checkstyle:HideUtilityClassConstructor") // Lombok generates private ctor.
+public class Constants {
```

Comment:
Move `Constants` to `opba-facade-protocol-api-shared`

Reply:
done